mlflow/utils/rest_utils.py

```diff
@@ -107,6 +107,8 @@ def http_request(
             f" To increase the timeout, set the environment variable {MLFLOW_HTTP_REQUEST_TIMEOUT}"
             " to a larger value."
         )
+    except requests.exceptions.InvalidURL as iu:
+        raise InvalidUrlException(f"Invalid url: {url}\n{iu}")
```

Comment:
```suggestion
        raise InvalidUrlException(f"Invalid url: {url}") from iu
```

Can we use `from` here?

Reply:
It is a good idea. It prevents duplicate URL information.

<img width="886" alt="截圖 2023-06-05 下午12 47 58" src="https://github.com/mlflow/mlflow/assets/13614761/4df2db99-1415-4234-b1c2-63bab40b022c">